ddtrace/tracer/tracer.go

```diff
@@ -130,6 +134,13 @@ const payloadQueueSize = 1000
 
 func newUnstartedTracer(opts ...StartOption) *tracer {
 	c := newConfig(opts...)
+	envRules, err := samplingRulesFromEnv()
+	if err != nil {
+		log.Warn("DIAGNOSTICS Error(s) parsing DD_TRACE_SAMPLING_RULES: %s", err)
```

Comment:
Totally feel like debug mode.

Reply:
I think warning level is fine, since these are errors that will prevent customer-specified sampling rules from working.